portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACUtils.macro

```diff
@@ -444,7 +444,7 @@ definition {
 
 		// It needs a 30 second pause which is the time required for the command to be executed. For more information about pause see LRAC-9518
 
-		Pause(locator1 = "30000");
+		ACUtils.sessionPause();
```

Comment:
Hey @calvinctvo, I don't think it's necessary to replace this macro with a macro that will only have a pause inside it. I think this would be necessary if the pause was in a .testcase, but since it's in a macro file you can leave the pause. What do you think?